src/main/kotlin/brs/services/impl/PeerServiceImpl.kt

```diff
@@ -610,7 +610,7 @@ class PeerServiceImpl(private val dp: DependencyProvider) : PeerService {
             when {
                 !beingProcessed.contains(peer) -> {
                     beingProcessed.add(peer)
-                    dp.taskSchedulerService.run { feedPeer(peer, foodDispenser, doneFeedingLog) }
+                    dp.taskSchedulerService.async(TaskType.IO) { feedPeer(peer, foodDispenser, doneFeedingLog) }
```

Comment:
If a sync *run* is used here, the broadcast transaction usually time out. The original code already had an *async* run here.